app/build.gradle

```diff
@@ -63,6 +63,7 @@ android {
             }
         }
         debug {
+            minifyEnabled false
```

Comment:
we found this change to help make the debugger a little more reliable...breakpoints weren't hitting consistently, but something else we can do is upgrade AS 😁